Source/Core/Core/PowerPC/JitArm32/JitAsm.cpp

```diff
@@ -424,8 +423,32 @@ void JitArmAsmRoutineManager::GenerateCommon()
 
 		POP(5, R0, R1, R2, R3, _PC);
 	}
+	const u8* storePairedS8 = GetCodePtr();
+	{
+		// R10 is the addr
+		// R11 is the scale
+		// R12 is scratch
+		// S0, S1 is the values
+		PUSH(5, R0, R1, R2, R3, _LR);
+
+		MOVI2R(R12, (u32)&m_quantizeTableS);
+		ADD(R12, R12, R11);
+		VLDR(S2, R12, 0);
+		VMUL(S0, S0, S2);
+		VMUL(S1, S1, S2);
```

Comment:
Could definitely use NEON here instead of VFP, but from a purely bugfix perspective this is fine.